google/datalab/contrib/bigquery/commands/_bigquery.py

```diff
@@ -147,22 +149,19 @@ def _get_execute_parameters(load_task_config_name, bq_pipeline_transformation_co
     # The name of query for execution; if absent, we return None as we assume that there is
     # no query to execute
     if 'query' not in bq_pipeline_transformation_config:
-      if any(key in bq_pipeline_transformation_config for key in ['large', 'mode']):
-        raise Exception('Query is not specified, but at least one query option is.')
+      # TODO(rajivpb): Log this: 'Query is not specified, but at least one query option is.')
```

Comment:
Why log this if the check for query options is removed?

Reply:
Removed the TODO now; changed my mind about needing a log statement here since if the user does not specify a 'query' key in the 'transformation' section, it's pretty clear that no transformation will be performed.